calculator/src/Components/Output.js

```diff
@@ -0,0 +1,12 @@
+import React, { Component } from "react";
+
+class Output extends Component {
+    render() {
+        return (
+            <div className="result">
+                <input className="inputscreen" placeholder="0" value={this.props.result} />
```

Comment:
I would suggest using .toLocaleString() here so that you can get the commas in your output